### DISCUSSION TITLE
Failed to load libsycl-native-bfloat16.spvException

### DISCUSSION THREAD

**Discussion by bli1348** [ggml-org/llama.cpp#17847] [Q&A]
I was running b7229 windows SYCL backend fine, but when I updated it to b7311 or b7313 I have the following error when running llama-server
Failed to load libsycl-native-bfloat16.spvException caught at file:D:\a\llama.cpp\llama.cpp\ggml\src\ggml-sycl\ggml-sycl.cpp, line:2997

**Comment by NeoZhangJianyu** [comment]
@bli1348 
I think there is missed so file for bflat16 support!
I will fix it as soon!

Could you share the GGUF model used in your case?

Thank you!

**Comment by bli1348** [comment]
b7349 works like b7229 now, but got the following error when running ministral3-14b-instruct-2512-ud-q4_k_xl from unsloth

srv  params_from_: Chat format: peg-native
slot get_availabl: id  3 | task -1 | selected slot by LCP similarity, sim_best = 1.000 (> 0.100 thold), f_keep = 0.289
slot launch_slot_: id  3 | task -1 | sampler chain: logits -> logit-bias -> penalties -> dry -> top-n-sigma -> top-k -> typical -> top-p -> min-p -> xtc -> temp-ext -> dist
slot launch_slot_: id  3 | task 1356 | processing task
slot update_slots: id  3 | task 1356 | new prompt, n_ctx_slot = 8192, n_keep = 0, task.n_tokens = 550
slot update_slots: id  3 | task 1356 | need to evaluate at least 1 token for each active slot (n_past = 550, task.n_tokens() = 550)
slot update_slots: id  3 | task 1356 | n_past was set to 549
slot update_slots: id  3 | task 1356 | n_tokens = 549, memory_seq_rm [549, end)
slot update_slots: id  3 | task 1356 | prompt processing progress, n_tokens = 550, batch.n_tokens = 1, progress = 1.000000
slot update_slots: id  3 | task 1356 | prompt done, n_tokens = 550, batch.n_tokens = 1
Native API failed. Native API returns: 20 (UR_RESULT_ERROR_DEVICE_LOST)
Exception caught at file:D:\a\llama.cpp\llama.cpp\ggml\src\ggml-sycl\ggml-sycl.cpp, line:426, func:operator()
SYCL error: CHECK_TRY_ERROR((*stream).memcpy((char *) tensor->data + offset, data, size).wait()): Exception caught in this line of code.
  in function ggml_backend_sycl_buffer_set_tensor at D:\a\llama.cpp\llama.cpp\ggml\src\ggml-sycl\ggml-sycl.cpp:426
D:\a\llama.cpp\llama.cpp\ggml\src\ggml-sycl\..\ggml-sycl\common.hpp:126: SYCL error

**Comment by NeoZhangJianyu** [comment]
I try b7351, it's passed.

GGUF: https://huggingface.co/unsloth/Ministral-3-14B-Instruct-2512-GGUF/resolve/main/Ministral-3-14B-Instruct-2512-UD-Q4_K_XL.gguf

Here is the cmd:
```
llama-cli.exe -m Ministral-3-14B-Instruct-2512-UD-Q4_K_XL.gguf -ngl 99 -c 4096 -p "who are you?"
```
